### PR TITLE
change code as per comments

### PR DESCRIPTION
**Pull request by 22PoojaGaur**
@uosl 

This is PR for #23 

- Made a separate archive.py to handle archive tasks
- Use envvars to get aws access information
- exit if envvars not set
- call upload_archives in docker.build

### CODE REVIEW

File: intermine_boot/docker.py [hunk 1]
```diff
@@ -152,90 +142,90 @@ def create_archives(options, env):
...
-        print(error)
-
-def download_archives_aws(options, env):
-    compose_path = _get_compose_path(options, env)
-    data_dir = compose_path.parent / 'data'
-    postgres_data_dir = compose_path.parent / 'data' / 'postgres'
-    solr_data_dir = compose_path.parent / 'data' / 'solr'
-    mine_data_dir = compose_path.parent / 'data' / 'mine'
-    version = generate_version(options, env)
+# def lsremote(url):
```
Comment (heralden) [inline intermine_boot/docker.py]: Could you delete it from the file as well? I don't see any reason to leave it commented out.

**Review by heralden — CHANGES_REQUESTED** [review]
Looks good! :+1: 

I have added one comment that we should fix before merging.

Also for the future, it would be great if you could create multiple commits with descriptive messages (*change code as per comments* won't make sense to anyone reading the commit log in the future :smile:). Your summary of 4 points would work great as the messages of 4 separate commits.
You don't need to change this now; just keep it in mind next time you commit.

File: intermine_boot/docker.py (hunk 1 above)
Comment (22PoojaGaur) [reply intermine_boot/docker.py]: deleted

**Comment by 22PoojaGaur** [comment]
@uosl I will keep the point regarding commit message in mind :smile:

**Review by heralden — APPROVED** [review]
Thanks!

**Pull request merged**